packages/proto-signing/src/directsecp256k1wallet.spec.ts

```diff
@@ -54,15 +57,30 @@ describe("DirectSecp256k1Wallet", () => {
     });
   });
 
-  describe("sign", () => {
+  describe("signDirect", () => {
     it("resolves to valid signature", async () => {
-      const wallet = await DirectSecp256k1Wallet.fromMnemonic(defaultMnemonic);
-      const message = toAscii("foo bar");
-      const signature = await wallet.sign(defaultAddress, message);
+      const { sequence, bodyBytes } = testVectors[1];
+      const wallet = await DirectSecp256k1Wallet.fromMnemonic(faucet.mnemonic);
+      const pubkey = {
+        typeUrl: "/cosmos.crypto.secp256k1.PubKey",
+        value: fromBase64(faucet.pubkey.value),
+      };
+      const fee = coins(2000, "ucosm");
+      const gasLimit = 200000;
+      const chainId = "simd-testing";
+      const accountNumber = 1;
+      const signDoc = makeSignDoc(
+        fromHex(bodyBytes),
+        makeAuthInfoBytes([pubkey], fee, gasLimit, sequence),
+        chainId,
+        accountNumber,
+      );
+      const signDocBytes = makeSignBytes(signDoc);
+      const signResponse = await wallet.signDirect(faucet.address, signDoc);
```

Comment:
In Lauchpad, we always use destruction here, like

```suggestion
      const { signature } = await wallet.signDirect(faucet.address, signDoc);
```

I think we can do this as well for all `signDirect` calls